code/script.js

```diff
@@ -253,45 +283,98 @@ const setSecret = () => {
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  generateBoard()
+  setSecret()
 }
 
 // setting the currentQuestion object when you select something in the dropdown
 const selectQuestion = () => {
   const category = questions.options[questions.selectedIndex].parentNode.label
   // This variable stores what option group (category) the question belongs to.
-  // We also need a variable that stores the actual value of the question we've selected.
+  const value = questions.options[questions.selectedIndex].value
 
   if (category === 'hair color') {
     currentQuestion = {
       attribute: 'hairColor',
-      // value: ,
-      // 👆 add the value from the input here
-      category: category,
+      value: value,
+      category: category
     }
   } else if (category === 'eye color') {
-    // Set this up your self
+    currentQuestion = {
+      attribute: "eyeColor",
+      value: value,
+      category: category,
+    }
+  } else if (category === 'character') {
+    currentQuestion = {
+      attribute: "character",
+      value: value,
+      category: category,
+    }
   } else if (category === 'accessories') {
     currentQuestion = {
-      //attribute: ,
+      attribute: value,      
       // 👆 this is the property of the booleans such as smoke, glasses and hat. add the value from the input here
       value: true, // we're asking if this person wears a hat for exaple, so always true in the question.
       category: category,
     }
   } else if (category === 'other') {
-    // Set this up your self (should be same structure as above)
+    currentQuestion = {
+      attribute: value,
+      value: true,
+      category: category,
+    }
+  }
+}
+
+const countingGuesses = () => {
+  //numGuess += 1
+  infoText.innerHTML = `
+  number of guesses: ${numGuess}
+  `
+  if (numGuess === 3) {
+    alert(
+      "You have made 3 guesses, now you must choose the person"
```

Comment:
Sneaky 🦊 I didn't realize I have only 3 shots when I was playing for the first time 😸  It makes your game even better 😻